lib/postgrex/replication.ex

```diff
@@ -542,7 +556,7 @@ defmodule Postgrex.Replication do
       it is not needed anymore. See `create_slot/4` for more details.
       Defaults to `true`.
 
-    * `:slot_opts` - Keyword list of options to create the slot with.
+    * `:slot_opts` - Keyword list of options used to create the slot.
       Only the keys `:slot_name`, `:temporary` and `:plugin` are allowed.
       By default, a temporary slot with the `:pgoutput` plugin is created.
       See `create_slot/4` for more details.
```

Comment:
Isn't it confusing that we have different defaults here? Does `copy_table` only make sense for temporarily slots?

Reply:
`Does copy_table only make sense for temporarily slots?`

No you're completely right, this is confusing. Addressed this in your other comment.